app/templates/components/mesh-manager.hbs

```diff
@@ -13,13 +13,16 @@
     <li>{{t 'general.currentlySearchingPrompt'}}</li>
   </ul>
 {{else}}
-  {{#if sortedSearchResults.length}}
+  {{#if searchResults.length}}
     <div class='selectable-list'>
       <ul class='mesh-search-results'>
-        {{#each sortedSearchResults as |term|}}
+        {{#each searchResults as |term|}}
           <li class="{{unless term.isActive 'disabled'}}" {{action 'add' term}}>{{term.name}}</li>
         {{/each}}
       </ul>
+      {{#if hasMoreSearchResults }}
+          <button {{action 'searchMore' query}}>{{t 'general.showMore' }}</button>
```

Comment:
Why pass query here as opposed to just using this.get('query') inside of the searchMore action?